src/bootstrap/suggest.rs

```diff
@@ -0,0 +1,71 @@
+use std::ops::Deref;
+use std::{path::Path, str::FromStr};
+
+use std::path::PathBuf;
+
+use crate::{
+    builder::{Builder, Kind},
+    tool::Tool,
+};
+
+/// Suggests a list of possible `x.py` commands to run based on modified files in branch.
+pub fn suggest(builder: &Builder<'_>, run: bool) {
+    let suggestions =
+        builder.tool_cmd(Tool::SuggestTests).output().expect("failed to run `suggest-tests` tool");
+
+    if !suggestions.status.success() {
+        println!("failed to run `suggest-tests` tool ({})", suggestions.status);
+        println!(
+            "`suggest_tests` stdout:\n{}`suggest_tests` stderr:\n{}",
+            String::from_utf8(suggestions.stdout).unwrap(),
+            String::from_utf8(suggestions.stderr).unwrap()
+        );
+        return;
```

Comment:
This should fail the build.
```suggestion
        panic!("failed to run `suggest-tests`");
```